src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationServiceImpl;
+    private static StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User userTest;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationServiceImpl = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        userTest = new User();
+        userTest.setLogin("userTest");
+        userTest.setAge(20);
+        userTest.setPassword("123456");
+    }
+
+    @Test
+    void register_userNull_notOk() {
+        assertThrows(RuntimeException.class, () ->
+                registrationServiceImpl.register(null));
+    }
+
+    @Test
+    void register_loginNull_notOk() {
+        User currentUser = userTest;

```

Comment:
unnecessary statement "User currentUser = userTest;"
you can just set up userTest.setLpgin(null)